README.md

```diff
@@ -1,3 +1,5 @@
+![build](https://github.com/ORNL/TASMANIAN/workflows/build-ubuntu/badge.svg?branch=master)
```

Comment:
Do we need this one here?

The documentation and python-pip scripts read from the README.md and make temporary changes, I would have to double-check that we are not interfering with those.

Reply:
I'll need to make couple of changes after we merge this, but it will be OK.